src/util.ts

```diff
@@ -0,0 +1,31 @@
+import { MessageEmbed } from "discord.js";
+import { UserKarmaInfo } from "./types";
+
+export function getEmbedFromLeaderboardList(usersKarma: Record<string, number>): MessageEmbed {
+    const embed = new MessageEmbed();
+    const list = Object.keys(usersKarma).map((userID, idx) => {
+        const rank = idx + 1;
+        const karma = usersKarma[userID];
+        return `**${rank}.** <@${userID}>   **${karma}** points`;
+    })
+    const desc = list.join('\n\n');
+    embed.setDescription(desc);
+    embed.setTitle('Leaderboard')
+    embed.setFooter('Leaderboad');
+    return embed;
+}
+
+export function getEmbedFromUserKarma(userKarma: UserKarmaInfo[]) {
+    const embed = new MessageEmbed();
+    const list = userKarma.map((userKarmaInfo, idx) => {
+        const rank = idx + 1;
+        const karma = userKarmaInfo.karma;
+        const channelID = userKarmaInfo.channel.id
+        return `**${rank}.**  <#${channelID}>        **${karma}**`;
```

Comment:
Why are we displaying ID of channel in user-facing output?

Reply:
Channel ID won't be shown as text, but `#<channel_name>` as link